packages/angular/ssr/src/app-engine.ts

```diff
@@ -124,7 +127,9 @@ export class AngularAppEngine {
       return undefined;
     }
 
-    const entryPointExports = await entryPoint();
+    const eventualExports = entryPoint();
```

Comment:
There's _technically_ still a race condition here if `entryPoint()` causes another call to `getEntryPointExports`. Not familiar enough with the code flow to know if that's a real concern, I suspect not.

Reply:
Tbh, it's not that much of a concern, although I am not against this improvement. I suggested a couple of things on slack, let me know what you think otherwise looks good.